### PR TITLE
Add relevant papers

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by NullHypothesis** [comment]
Hey Vasilis, I hope you've been well! Thanks for this PR. My unwritten policy has been to only add work that has been peer-reviewed by well-known CS venues. It's not a perfect policy but it strikes me as better than the alternative of erring on the side of completion: There are lots of technical reports out there and I cannot keep up with adding them all, let alone keeping track of revisions. That said, Ververis2021a does seem like a great addition. Would you mind rebasing onto master and formatting the bibtex record like all the others?

**Review by NullHypothesis — CHANGES_REQUESTED** [review]
See this comment: https://github.com/NullHypothesis/censorbib/pull/16#issuecomment-1537182376

File: references.bib [hunk 1]
```diff
@@ -1,3 +1,12 @@
+@inproceedings{Ververis2021a,
+       title        = {Understanding Internet Censorship in Europe: The Case of Spain},
```
Comment (NullHypothesis) [inline references.bib:2]:
```suggestion
       title        = {Understanding {Internet} Censorship in {Europe}: The Case of {Spain}},
```

File: references.bib [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+       author       = {Vasilis Ververis and Tatiana Ermakova and Marios Isaakidis and Simone Basso and Benjamin Fabian and Stefania Milan},
+       year         = {2021},
+       booktitle    = {13th ACM Web Science Conference 2021},
+       publisher    = {Association for Computing Machinery},
```
Comment (NullHypothesis) [inline references.bib:6]:
```suggestion
       publisher    = {ACM},
```

File: references.bib [hunk 3]
```diff
@@ -1,3 +1,12 @@
+@inproceedings{Ververis2021a,
+       title        = {Understanding Internet Censorship in Europe: The Case of Spain},
+       author       = {Vasilis Ververis and Tatiana Ermakova and Marios Isaakidis and Simone Basso and Benjamin Fabian and Stefania Milan},
+       year         = {2021},
+       booktitle    = {13th ACM Web Science Conference 2021},
```
Comment (NullHypothesis) [inline references.bib:5]:
```suggestion
       booktitle    = {Web Science Conference},
```

File: references.bib [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+       publisher    = {Association for Computing Machinery},
+       url          = {https://doi.org/10.1145/3447535.3462638},
```
Comment (NullHypothesis) [inline references.bib:7]:
```suggestion
       url          = {https://dl.acm.org/doi/pdf/10.1145/3447535.3462638},
```

**Review by NullHypothesis — CHANGES_REQUESTED** [review]
Made several fixes.

**Comment by anadahz** [comment]
> Made several fixes.

Thank you!

**Review by NullHypothesis — APPROVED** [review]
Looks good, thanks for your help! ❤️

**Pull request merged**